avro-serializer/src/test/java/io/confluent/kafka/serializers/KafkaAvroSerializerTest.java

```diff
@@ -46,7 +47,7 @@
 
   public KafkaAvroSerializerTest() {
     schemaRegistry = new LocalSchemaRegistryClient();
-    avroSerializer = new KafkaAvroSerializer(schemaRegistry);
+    avroSerializer = new KafkaAvroSerializer(schemaRegistry, GenericDatumWriter.class);
```

Comment:
Ok. Thought the constructor used for tests was supposed to take any objects you might want to inject mocks for, but I guess you can use `configure` for that too. I'll remove argument.